src/utils/ValidationHelper.ts

```diff
@@ -102,27 +102,85 @@ export class ValidationHelper {
 		return "";
 	};
 
-	isAddressFormatValid = (jwtPayload: JwtPayload): { errorMessage: string; errorMessageCode: string } => {
-		const personIdentityAddresses: PersonIdentityAddress[] = jwtPayload.shared_claims.address;
-		for (const address of personIdentityAddresses) {
-			if (!this.checkIfValidCountryCode(address.addressCountry)) {
-				return {
-					errorMessage: "Invalid country code: country code is not GB in the postalAddress",
-					errorMessageCode: MessageCodes.INVALID_COUNTRY_CODE,
-				};
-			} else if (!this.checkIfAddressIsValid(address)) {
-				// Validation fails if all the mandatory postalAddress fields- subBuildingName, buildingName, buildingNumber and streetName are missing and is not a valid string or if all the 3 mandatory fields- subBuildingName, buildingName, buildingNumber are missing or not a valid string
-				return {
-					errorMessage: "Missing all or some of mandatory postalAddress fields (subBuildingName, buildingName, buildingNumber and streetName), unable to create the session",
-					errorMessageCode: MessageCodes.MISSING_ALL_MANDATORY_POSTAL_ADDRESS_FIELDS,
-				};
-			}
+	isSharedClaimDataValid(jwtPayload: JwtPayload): { errorMsg: string; errorMsgCode: string } {
+		// Validate user details like givenName, familyName and email
+		if (!this.checkIfValidString([jwtPayload.shared_claims.emailAddress])) {
+			return {
+				errorMsg: "Missing emailAddress from shared claims data",
+				errorMsgCode: MessageCodes.MISSING_PERSON_EMAIL_ADDRESS,
+			};
+		} else if (!jwtPayload.shared_claims.name || !this.isPersonNameValid(jwtPayload.shared_claims.name)) {
+			return {
+				errorMsg: "Missing person's GivenName or FamilyName from shared claims data",
+				errorMsgCode: MessageCodes.MISSING_PERSON_IDENTITY_NAME,
+			};
 		}
 		return {
-			errorMessage: "",
-			errorMessageCode: "",
-		};
-	};
+			errorMsg: "",
+			errorMsgCode: "",
+		};		
+	}
+
+	isPersonNameValid(personName: PersonIdentityName[]) : boolean {		
+		let isValid = true;
+
+		if ( personName.length === 0 ) {
```

Comment:
I always run lint:fix and lint before i raise a PR. It looks fine on IDE, just doesnt look right here in the PR.

Reply:
Hm because it's showing like that in my IDE 
<img width="506" alt="image" src="https://github.com/alphagov/di-ipv-cri-f2f-api/assets/40401118/eb5c72ea-2a67-4e7a-b754-f9dad1b22974">
We also have a trailing tab on line 124

Wonder if we need to look at the rules again